watson_developer_cloud/watson_service.py

```diff
@@ -242,6 +281,23 @@ def set_api_key(self, api_key):
         self.api_key = api_key
         self.jar = CookieJar()
 
+    def set_token_manager(self, iam_api_key, iam_access_token, iam_url):
+        if iam_api_key == 'YOUR IAM API KEY':
```

Comment:
I am confused by the string `'YOUR IAM API KEY'`. Do you mind explaining?

Reply:
This is the way the python tests were initially set up for integration tests. So, for example if it gets the username="YOUR SERVICE USERNAME", it will set it as None so that later the `.env` variables can be set.

Reply:
Yeah, agree